tests/virtual_machines_test.go

```diff
@@ -41,44 +44,409 @@ var _ = Describe("Virtual Machines", func() {
 	Context("Check the client", func() {
 		AfterEach(func() {
 			vmList := &kubevirtv1.VirtualMachineList{}
-			err := testClient.VirtClient.List(context.TODO(),&client.ListOptions{},vmList)
+			err := testClient.VirtClient.List(context.TODO(), &client.ListOptions{}, vmList)
 			Expect(err).ToNot(HaveOccurred())
 
 			for _, vmObject := range vmList.Items {
-				err = testClient.VirtClient.Delete(context.TODO(),&vmObject)
+				err = testClient.VirtClient.Delete(context.TODO(), &vmObject)
 				Expect(err).ToNot(HaveOccurred())
 			}
```

Comment:
I think it will be nice to add here a something like this

```
Eventually(func() int {
				vmList := &kubevirtv1.VirtualMachineList{}
				err := testClient.VirtClient.List(context.TODO(), &client.ListOptions{}, vmList)
				Expect(err).ToNot(HaveOccurred())

				return len(vmList.Items)

			}, 30*time.Second, 3*time.Second).Should(Equal(0),"failed to remove all vm objects")
```